### ISSUE TITLE
how to change the icon?

### ISSUE THREAD

**Issue by vieyama** [saltyshiomix/nextron#52]
i want to ask, how to change the icon?

**Comment by saltyshiomix** [comment]
Hi @vieyama !

Just replace `resources/icon.icns` and `resources/icon.ico` 👍 

If you have any troubles, it may be the cache problem which electron does.

**Comment by saltyshiomix** [comment]
No any updates, so I closed it :)

**Comment by Ngomana** [comment]
I'm unable to change the icon nor add images to my app

**Comment by Psycokwet** [comment]
Did you try the solution mentionned earlier ?

You can add images by putting them into renderer/public folder

**Comment by mittnavnermike** [comment]
So I struggled for a while and finally figured it out. I thought I'd share.

It was working just fine until I added `electron-updater`. This process made me add some build config in my `package.json` for distribution. This meant that part of my config from `electron-builder.yml` was ignored. I added the mac->icon part of the config to `package.json` under the build key, which did the trick.

I am sure I could just move all that config to `electron-builder.yml` tho.

My `package.json` ended up looking like this:
```json
{
...,
 "build": {
    "mac": {
      "icon": "resources/icon.png"
    },
    "buildDependenciesFromSource": true,
    "publish": [... ]
  },
}
```

Hope this helps someone
